product_warranty/models/product_warranty.py

```diff
@@ -0,0 +1,131 @@
+# -*- coding: utf-8 -*-
+# ########################################################################
+#                                                                        #
+#                                                                        #
+# ########################################################################
+#                                                                        #
+# Copyright 2015 Vauxoo
+# Copyright (C) 2009-2011  Akretion, Emmanuel Samyn, Benoît Guillot      #
+#                                                                        #
+# This program is free software: you can redistribute it and/or modify   #
+# it under the terms of the GNU General Public License as published by   #
+# the Free Software Foundation, either version 3 of the License, or      #
+# (at your option) any later version.                                    #
+#                                                                        #
+# This program is distributed in the hope that it will be useful,        #
+# but WITHOUT ANY WARRANTY; without even the implied warranty of         #
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the          #
+# GNU General Public License for more details.                           #
+#                                                                        #
+# You should have received a copy of the GNU General Public License      #
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.  #
+##########################################################################
+
+from openerp import api, fields, models
+
+
+class ReturnInstruction(models.Model):
+
+    _name = "return.instruction"
+    _description = "Instructions for product return"
+
+    name = fields.Char('Title', required=True)
+
+    instructions = fields.Text(
+        'Instructions',
+        help="Instructions for product return")
+
+    is_default = fields.Boolean('Is default',
```

Comment:
Align this way to avoid a lot of lines:
```
is_default = fields.Boolean(
    'Is default', 
    help="If is default, will be use to set the default value in supplier "
         "infos. Be careful to have only one default")
```